camera:
  image_flip: true
  camera_position: 0.8
  fx: 476.398
  fy: 476.397
  cx: 320.279
  cy: 245.871
  roll: 0.07
  pitch: -0.05
  yaw: -0.22
  T1: 0
  T2: 0
  T3: 0

